README.md

```diff
@@ -1838,6 +1838,8 @@ ANSWERS.addComponent('Map', {
   apiKey: '',
   // Optional, can be used for Google Maps in place of the API key
   clientId: '',
+  // Optional, used to determine the language of the map
```

Comment:
Could the default value be listed here in the README comment as well?

Reply:
Sure